src/components/allComments.js

```diff
@@ -38,6 +39,33 @@ class AllComments extends React.Component {
     } else {
       this.setState({ loading: false, isSignedIn: false })
     }
+    this.loadUserOwnerComments()
```

Comment:
With this we can get rid of the sign-in check all together, can't we?

Reply:
Yes we can get rid of it if you don't need the sign in there since it's no use for public comments